pyrakoon/compat.py

```diff
@@ -173,7 +173,7 @@ def wrapped(*args, **kwargs):
     
 
 class ArakoonClient(object):
-    def __init__(self, config):
+    def __init__(self, config, timeout=0, noMasterTimeout=0):
```

Comment:
Name the new parameters in the docstring
Whats the difference between `timeout` and `noMasterTimeout`